src/obb/server/controllers/index.cljc

```diff
@@ -0,0 +1,9 @@
+(ns ^{:added "0.1.0" :author "Pedro Pereira Santos"}
+  obb.server.controllers.index
+  "Controller to handle the root/index request to the server")
+
+(defn handler
+  "Returns a intro to the game"
+  [request]
+  {:body "Hello World!"
+   :status 200})
```

Comment:
Ok that sounds good.

On clan we have 3 layers:
* The core stuff - logic, something like interactors but not only for use cases. It's namespaces that only know clojure and perform actions
* The controllers - mainly convert data to and from the core. Typically get the necessary data from the http request, call core, and then generate the http response.
* The dal - indirection over the datastore, used by the core

I prefer handlers to http-handlers, because we can have handlers for non-http stuff (nothing comes to mind, but...). Ok to refactor this to handlers?